cdist/exec/local.py

```diff
@@ -238,7 +239,7 @@ def run_script(self, script, env=None, return_output=False,
                         message_prefix=message_prefix)
 
     def save_cache(self):
-        destination = os.path.join(self.cache_path, self.hostdir)
+        destination = os.path.join(self.cache_path, self.target_host)
```

Comment:
@asteven Well, it would be more consistent. And if I am right there is a plan to fully implement cache functionality and then I think this will matter.